test/test_reload.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+servers_list = [13301, 13302]
+
+def test_reload_fast(servers, helpers):
+    assert servers[13301].probe_uri('localhost:13302') is True
+    assert servers[13302].get_member('localhost:13301')['status'] == 'alive'
+
+    assert servers[13301].conn.eval('return package.reload()') == [True]
+
+
+def test_reload_slow(servers, helpers):
```

Comment:
I think it would be great to test some basic scenarios here. Such event subscription, status change, etc.